pygenpass/database.py

```diff
@@ -40,18 +42,23 @@ def __init__(self):
 
     def insert_data(self, portal_name, password, creation_date, email, portal_url):
         """Adding values into database"""
-        self.password = password
-        self.creation_date = creation_date
-        self.email = email
-        self.portal_name = portal_name
-        self.portal_url = portal_url
-        self.cursor_obj.execute(
-            """INSERT INTO passwords
-            (portal_name, password, creation_date, email, portal_url)
-            VALUES (?, ?, ?, ?, ?)""",
-            (self.portal_name, self.password, self.creation_date, self.email, self.portal_url),
-        )
-        self.con.commit()
+        try:
+            self.password = password
+            self.creation_date = creation_date
+            self.email = email
+            self.portal_name = portal_name
+            self.portal_url = portal_url
+            self.cursor_obj.execute(
```

Comment:
@mayurilahane Why do we need variable initialization inside the try block?

Reply:
yes, agree!
we can do this try where we are actually getting values.
thanks for suggesting!